common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -286,9 +286,6 @@ def test_problem_type_a11y(self):
 
         self.problem_page.a11y_audit.config.set_rules({
             "ignore": [
-                'aria-allowed-attr',  # TODO: AC-491
-                'aria-valid-attr',  # TODO: AC-491
-                'aria-roles',  # TODO: AC-491
                 'checkboxgroup',  # TODO: AC-491
```

Comment:
Created https://openedx.atlassian.net/browse/TNL-5576 for checkboxgroup and radiogroup issues.